File: client/src/app/_models/order.ts
export class Orders {
  OrderNo: number;
  CustomerId: number;
  CustomerName: string;
  OrderDate: Date;
  ManufacturerId: Number;
  ManufacturerName: string;
  OrderedByName: string;
  Fulfilled: boolean;
  OrderLine: OrderLine[];
}

export class OrderLine {
  OrderNo: number;
  LineNo: number;
  LocationId: number;
  MachineId: number;
  Capacity: number;
  CostPerUnit: number;
  LineAmount: number;
  CapacityEntryNo: number;
}

File: API/Models/Order.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace API.Models
{
    public partial class Order
    {
        public Order()
        {
            OrderLines = new HashSet<OrderLine>();
        }

        public int OrderNo { get; set; }
        public int CustomerId { get; set; }
        public string CustomerName { get; set; }
        public DateTime OrderDate { get; set; }
        public int ManufacturerId { get; set; }
        public string ManufacturerName { get; set; }
        public string OrderedByName { get; set; }
        public bool Fulfilled { get; set; }

        public virtual Customer Customer { get; set; }
        public virtual Manufacturer Manufacturer { get; set; }
        public virtual CustomerInvoice CustomerInvoice { get; set; }
        public virtual ICollection<OrderLine> OrderLines { get; set; }
    }
}
